resources/zeebe.json

```diff
@@ -315,7 +315,7 @@
     {
       "name": "CalledDecision",
       "superClass": [
-        "Element"
+        "BindingTypeSupported"
```

Comment:
[not blocking] Are we going to use the parent type information anywhere? If not, I'd rather expect to use `extends` property on the type defining the bindingType.

Reply:
@barmac I made the change you suggested. Makes much more sense. Good catch.

Reply:
Exactly.